### DISCUSSION TITLE
希望可以增加支持本地部署+本地文件OneDrive的版本

### DISCUSSION THREAD

**Discussion by samleong123** [spencerwooo/onedrive-vercel-index#289] [Ideas]
您好，
感谢您开创的这个Repo , 我个人很喜欢您做的UI , 应该是 OneDrive index 里面目前我看过最美的
希望可以增加如下支持：
1. 在VPS上自己部署
2. 可以支持本地文件检索功能 (类似Directory Index / h5ai 这类本地目录)

**Comment by spencerwooo** [comment]
1. 在 VPS 上部署可以用 docker 部署，之前有同学贡献了 dockerfile：https://github.com/spencerwooo/onedrive-vercel-index/discussions/112
2. 就像不会支持 Google Drive 等其他网盘一样，本项目只针对 OneDrive，主要维护在 Vercel 上的部署，我个人精力有限，其它环境都大概率不会支持
